packages/vscode-extension/src/common/Project.ts

```diff
@@ -101,10 +101,18 @@ export type TouchPoint = {
   yRatio: number;
 };
 
+export enum ActivateDeviceResult {
+  succeeded,
+  notEnoughSeats,
+  unableToVerify,
+  connectionFailed,
```

Comment:
I still don't think thats and exhausting list, or rather maybe doesn't cover some important aspects of verification.

There should be a clear state indicating the key is invalid, e.g. keyVerificationFailed which should be used when we are certain that this is the reason the request failed (perhaps we'd get a specific http return code in this case).

Using `unableToVerify` as a wildecard for other types of errors is good, but you should keep in mind when it happens the message to the user should be different than in the case they put in an invalid license key. Specifically the issue could be that our servers are down and we get 500 errors from load balancer. This will still not cause fetch to throw